keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/jackson/mixins/ReferenceRuleMixin.kt

```diff
@@ -0,0 +1,8 @@
+package com.netflix.spinnaker.keel.ec2.jackson.mixins
+
+import com.fasterxml.jackson.annotation.JacksonInject
+
+internal interface ReferenceRuleMixin {
+  @get:JacksonInject("name")
+  val name: String
```

Comment:
This is how the parsed `ReferenceRule` will pick up the injected `name` if one is not supplied